javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -22,15 +24,18 @@
  * @author Ruslan Sennov
  * @since 2.0.0
  */
-public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T>, Serializable {
-
+public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T> {
```

Comment:
I see it is final again (can't comment on out-dated diffs). It is important because the return types cannot be adjusted by sub-classes of Vector any more. 

IndexedSeq has `Option<? extends IndexedSeq<T>> initOption();`

Vector has `Option<Vector<T>> initOption();`.

A subclass `X`cannot have `Option<X<T>>` any more.